### DISCUSSION TITLE
Default screen

### DISCUSSION THREAD

**Discussion by defispartan** [aave/interface#696] [General]
Currently the default screen when visiting the app is the Dashboard. There's pros and cons for defaulting to Dashboard or Markets.

**Dashboard**
Pros:
- The primary purpose of the app is to use rather than view, so putting people on a screen where they can most easily use the app best accommodates this.

Cons: 
- If a user doesn't have a wallet, they will be more interested in the markets data
- Particularly bad UX on mobile where users have to click top right icon to navigate off of dashboard

**Markets**
Pros:
- Better experience for users who don't have a wallet, or are new to the protocol

Cons:
- Users have to navigate to another screen to view and interact with their positions

One option is to change the default screen based on whether someone is currently connected (https://github.com/aave/interface/issues/660 ), or has connected before (can create localStorage variable) .

**Comment by sakulstra** [comment]
Agree with these points except for:
> Particularly bad UX on mobile where users have to click top right icon to navigate off of dashboard

which is not a correct argument - especially on mobile (thinking gnosis or coinbase dapp store) being on dashboard directly is an advantage so i guess it depends(agree that when just checking out a market being on the market directly is more helpful). Also for these cases we automatically connect the wallet afaik, so "redirecting when not connected" might not be trivial.
